samples/projection-management/Program.cs

```diff
@@ -0,0 +1,312 @@
+		}
+
+
+
+
+
+		private static Task Delete(EventStoreProjectionManagementClient managementClient) {
+			// not available 
+			return Task.CompletedTask;
+		}
+
+		private static async Task Abort(EventStoreProjectionManagementClient managementClient) {
+			
+			try {
+				var js =
+					"fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+				await managementClient.CreateContinuousAsync("countEvents_Abort", js);
+			} catch (InvalidOperationException e) when (e.Message.Contains("Conflict")) {
+				// ignore 
+			}
+			
+			
+			#region Abort
+			// The .net clients prior to version 21.6 had an incorrect behavior: they will save the checkpoint.
+			await managementClient.AbortAsync("countEvents_Abort");
+			// It is safe to call abort multiple times 
+			await managementClient.AbortAsync("countEvents_Abort");
+			#endregion Abort
+
+		}
+
+		private static async Task Reset(EventStoreProjectionManagementClient managementClient) {
+
+			try {
+				var js =
+					"fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+				await managementClient.CreateContinuousAsync("countEvents_Reset", js);
+			} catch (InvalidOperationException e) when (e.Message.Contains("Conflict")) {
+				// ignore 
+			}
+			
+			#region Reset
+			// Checkpoint will be written prior to resetting the projection
+			await managementClient.ResetAsync("countEvents_Reset");	
+			// It is safe to call abort multiple times 
+			await managementClient.ResetAsync("countEvents_Reset");
+			#endregion Reset
+
+		}
+
+		private static async Task CreateOneTime(EventStoreProjectionManagementClient managementClient) {
+
+			const string js =
+				"fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+			await managementClient.CreateOneTimeAsync(js);
+
+		}
+
+		private static async Task CreateContinuous(EventStoreProjectionManagementClient managementClient) {
+
+			#region CreateContinuous
+			try {
+				var js =
+					"fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+				await managementClient.CreateContinuousAsync("countEvents_Create", js);
+
+
+			} catch (InvalidOperationException e) when (e.Message.Contains("Conflict")) {
+				Console.WriteLine($"countEvents_Create already exists");
+			}
+			#endregion CreateContinuous
+
+
+		}
+
+
+
+		private static async Task Update(EventStoreProjectionManagementClient managementClient) {
+
+			#region Update
+			const string js = "fromAll().when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+			var name = $"countEvents_Update_{Guid.NewGuid()}";
+
+			await managementClient.CreateContinuousAsync(name, "fromAll().when()");
+			await managementClient.UpdateAsync(name, js);
+			
+			
+			try {
+				await managementClient.UpdateAsync("countEvents_Update_Not_existing", js);
```

Comment:
**CONCERN:** As [above](https://github.com/EventStore/EventStore-Client-Dotnet/pull/143/files#r686595897): we should have a separate section on the error handling as we have, e.g. for the append failures: https://developers.eventstore.com/clients/grpc/appending-events/#handling-concurrency